tests/components/dynalite/test_bridge.py

```diff
@@ -63,19 +62,8 @@ def dyn_bridge():
     device2 = Mock()
     device2.category = "switch"
     reg_func = Mock()
-    dyn_bridge.register_add_devices(reg_func)
+    # First register
+    dyn_bridge.register_add_devices("light", reg_func)
```

Comment:
Do the same things here as above but insert an extra await of `hass.async_block_till_done` between point 3 and 4 to make sure the platforms are set up before the devices are added.

Reply:
done